tests/integration/targets/vm_snapshot_info/tasks/main.yml

```diff
@@ -0,0 +1,48 @@
+---
+- environment:
+    SC_HOST: "{{ sc_host }}"
+    SC_USERNAME: "{{ sc_username }}"
+    SC_PASSWORD: "{{ sc_password }}"
+    SC_TIMEOUT: "{{ sc_timeout }}"
+
+
+  # TODO: update tests to work with other params - vm_name, serial
+  block:
+    # --------------------------------------------------------
+
+    - name: List all VM snapshots - API
```

Comment:
Good for now.

But I would like to have a dedicated task that will setup on server exactly number of VMs (like 2), with snapshots in exact state (like 3 manual/USER snapshots with unique label, and 3 snapshots with non-unique label). Than you can check if returned record contains correct values.

That setup-vms-and-snapshots-for-this-test task - it can be done in separate PR. File would go in `tests/integration/prepare/` . There is already a `prepare_iso.yml`, it uploads an ISO image, so that ISO integration test does not need to upload a large ISO each time (it can upload a very small ISO instead - it is much faster when testing over VPN).